web-ui/view/documentation.ml

```diff
@@ -22,18 +22,18 @@ let getting_started =
               p
                 [
                   txt
-                    "It uses metadata from the project's opam and dune files \
-                     to work out what to build, and uses caching to make \
-                     builds fast. It uses the information in the opam files in \
-                     the project to automatically test against multiple OCaml \
-                     versions and OS platforms.";
+                    "OCaml-CI uses metadata from the project's opam and dune \
```

Comment:
```suggestion
                    "OCaml-CI uses metadata from the project's opam and `dune` \
```

Reply:
The backticks do not result in the markup that we are hoping for -- leaving off for now.

Reply:
Bummer. If not, perhaps Dune should be capitalised as a proper noun if we cannot specify it in monospace for `dune`f files. In the Dune documentation, we differentiate the build system (Dune) by the command and file(s) `dune`.